.github/workflows/pages-deployment.yaml

```diff
@@ -19,6 +19,8 @@ jobs:
           cache: 'pip'
       - name: Install python dependencies
         run: pip install -r requirements.txt
+      - name: linkml gen-taxonomy
+        run: make -C schema gen-taxonomy
```

Comment:
Let's move this out of the githhub workflow and have geo-taxonomy be a make command that gets run locally. Maybe add it to the README.md as part of the contributing instructions that whenever you are adding a taxonomy term you need to link it to the link schema by running `make -C schema gen taxonomy` ?